plot_util.py

```diff
@@ -0,0 +1,62 @@
C0a6SsJm+q/BJGu+Pp8RwSeia3xfQNmvzKKjQ2+liwVaP9CLC6v9LIf4WOwUU2ltTMAFg/9VV7NL2hJU2aFBr0cDlxggU0zG8fRTAfjih40cTKiWURVQRqoCBVsEE6HinELtCArJm908aI8CpJvVBEnASvB1NS81g8cL6f0EQh960j4kpDnBoHHT6WhrYoImE9gUJsP513exi5LrklDPUzDiF3e0pAGDg51DOM4R0wHRnDsTDjUwBAkNJN4Lq985fujPM+J4nRQcyI4ztwVns1pcxu6NkUt4jcqI3UmPT3R4WfV9Hr2DSnrkbJxghrmOHl0Z/G3viLuSi6dEr/Ap36C+TtuYYEwcmraFU/wK/5ABvgYp5aN6qlWBZYxiTcFATf4oZHCKb0lW16+qAmxruwsqsOYkX0wmLreCMMXTa0mcDDusDvhD9Vl+gEyeEnb2SvMcrYR7mdD3geTi7X9MFZgD4LoXlEhQOd9BEVYHR0sSEJiw0JAgXkdErOAmq6YETFsmLroeaXuUebLUTkCvnY8WeYlSz4fYMYCY20IrXNQtCxqGpl9AEKZDmjCOXiRJ3Zo6RhExnW54xavwzF34nHUWZ4OjPR4/imvpe64JzZ5aH205XC8fICSNqcuk1PLhXLXYkNuy02BG7AOu/JsHdLz2Jd+2hzaGrpeBNqFxKGOjmNXeulU0iLBlQW3tsNmtGc70VPKEBZQUSw1sGxjW2o+yHE5LVMLZsYtwfmFC2XhwoVy4w7Zm2YGWB081lSeWrBMijH2andOR70IriMpA7LlCXukbdmLSUF909yfxwAeqRghQIDIgZRhhgdMAtKouC2QBw1HITo4xDcN4RFsXH1yzFsR8NbL4upS2X94fxRMTmnPwrWVWnOal+cGICDDpUJUtQlhPLRDuxzX8UMs9dHJNI7LOFhXb6bAwfW1UpoOmNNgt2nyVx7Gt6GjvzRn+sn9Dqbl6uKth6I5j8Zd3ZmijsvhjNI4b7pOi0CsUdKMv9RBh2tq65USboUzOO6wwOvaOa58tB9bWtLYxIy2My8a80efPllOnb1QbrrxRVpmrzGPoYMTeKpLCbeAlNO+4LOHB1CeCQDrnvEZkxorxwBVuB1KrY+FhSKe1vtHNg0lre8UAhTzcBJ6simgVJhCTKnC3GgRI4ep4Iq585Cz0as/DZZ1KVxF+bxACcFRP0sr+aUhQrPPQM4axFu7EK0I7cDRXFrMDAXRg71iGLvhZOGdJ2GBSJCMRjIoC+oZ4S4XRlFqPUR44PFBHQBC4ApnFI18lCl2GbKSiTKlU/GCkTSSMsIL1CgYnEhcQah1DGE2vIfgygxkrIlBvqFRcmACKo+7N+LKK0TXWHJ+uIoc4AMiF1JPEpLQoGG8UVEUJBbJEBB4gobORiVew9jrq+dWeaJjBPNUPjbIFZbJgmfUAVcq6Coxi4AYrF8MxpUPdMKAUWYhN6qMoMu4n9U2PRUQsr6SvHCxHD15pnzbX3l5Wbq4oO8dAFWZvMkEq37b8SAKfqfAFuhxGyk+tDSdgHqGB4csThL9OlsQ6i9sORpRyylTx61P5hfH8zcJemuwsc5XhtV5umeJQRhGh5BMF7JTc/UhM4dsXvxoIlHUSPlwCRAdpwqzu4WJ0UPPaBEcGvoWh4NbSSeBxhBQexCcXNJyy4uxqGhqak57qKWq+FAUzWYA2XFk5yBAlVkc41YmxIOnigbyKcEnlgESDwYYNJJW9cMv9Y384CdcRFEGK/xuzGBa5hAJN8o5BR6nhFemQqn8qEgh+SUeZThWxouwqEj/OoEnnUWsEzBaMeBNug2uE5jSrhNOUWvUDXMdpB27HDId6CdQAwc0+o8C2pn04NF/+Yl6CoMSJC+QS5oOXHGy3uAT2MoH78hEMvo66uq4Jgpn97eSQ21wRZJRs8mkbBiHfEkfTmJCV2O3P0h1imKdIja/lKs7g4f0QlLXhI4RhRoHOkc3APinH+jDRu7A1oNwXezYfQu6M+fOxV6dc3NzZXVFtweiiVrQffARCwVgYaOyNwycNEbNq/Z1XbBtp+AhQ2x4wqhBZuFFq0QTfYqn0dbFG9yXK0zPTFEsZlSqqmJg6kvExHGVF/IEXrQ2KDxNcIQ8W6wLupgrq4PG4Rj4m1Zj8V4xYCTZnD4HzaDf2KhTIKVX9Uxgx/yNZc8TR8qh3bujQctrCzBH1zSjD70Gnrca4N3YkX27yvy73qwfIC2U7frdwaRmRRsTq2E3zKAxdgyMlbeENb1hiCu9YNgaz+BiIoBzikUIaaDxXC0cJqYpe9QmRKxqlAUKLx37lGY3sR/IzTffqKtveheMmPeb4q16mVYk4crqYpmdnYnfAlBxLpGkMjzNRpTxxddVfb1lUvsbAKMUmhV5OLmRaMyMBGHjEV6XcGuA56OR8zPzZWP5ReXnf+bHy6JuKzbk3abEL3ypFMPOxmsbS4EfcqkOFmFEPaEMlceyTfFDeJoqOO1BDyiNXzb6apezBxwUDg5nhg9lg5bMh/JVBg9wQcGZrUkuygjhKKUrYg5ajNMTkVI1KBGdJ9jKajo/6H2AtbqyonvDtTKre0bu79wmmKyKJmYLopmdkS7Zycm8FSOXWNeQ76bpP2BsOMNeE528yE1/5QDINO1TXvxXJAcLuBgYlKFDymg3cqDLWX2Qg9kczts7W8UMrwqyIV1tTM7GpIsZA22xjsEPPanvmS1aZ9RL2lPYqUnkyL7hKsd4Qu/UAw/w4uKi/ka3tI96HMRauLq9nVbb6TsN+JiNCp76pW0xstQWy0ihwNGOfD4Vu2mL/4z07naEEamPZ2ZyXNDTy0tLUY4c6B7drGmrPTbdRWfwnVIZgfYg+9wscG3Eo1/aLmnfjXe++bvKS2/UA/JtLxA/9bkuoPHyXH01Nz+nb3yIp2i3bZsPHWRfMb4Zm2nIjJPQoWRCTl+4ZzXL0CDSfh+r6r/Zcv7CebJlfvtNkkhySdfY5P8H3O3dKpbtZ9QAAAAASUVORK5CYII="
```

Comment:
It'd be nice nice to know where `default_base64` came from, or how it's created.

I think https://stackoverflow.com/a/3715530 is useful. It worked for me:

``` python
In [1]: import base64

In [17]: with open("github.png", "rb") as f:
    ...:     s = base64.b64encode(f.read())
    ...:

In [17]: s[:10]
Out[17]: b'iVBORw0KGg'

```